src/main/java/luke/exception/DeleteIndexOutofboundsException.java

```diff
@@ -0,0 +1,7 @@
+package luke.exception;
+
+public class DeleteIndexOutofboundsException extends LukeException {
```

Comment:
Since you are using many different exception classes, I would suggest putting a default `msg` string instead of always taking it from the constructor.

So instead of doing
``` throw new DeleteIndexOutofboundsException("\n\tYou have typed in an invalid number.\n\tPlease check your list again.");```
in your code you can simply do this:
``` throw new DeleteIndexOutofboundsException();```
Also makes those sections easier to read.

Reply:
Yeah, I wasn't too familiar with using exception classes and I needed some help on this. Thanks for the suggestion 👍